android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1135,7 +1089,7 @@ public void showOnboardingV2(boolean fromStats) {
     }
 
     public void hideRewardsOnboardingIcon() {
-        BraveToolbarLayoutImpl layout = (BraveToolbarLayoutImpl) findViewById(R.id.toolbar);
+        BraveToolbarLayoutImpl layout = findViewById(R.id.toolbar);
```

Comment:
nit: This code is being repeated several times, should we make a helper method for this.

Reply:
I noticed that, we can and will but in a specific refactoring/cleanup PR, not in this as this is a pretty big file and it would take quite a time to test these clean up changes. I just cleaned minor things like this side by side, but didn't want to divert away.

Reply:
Created an issue [22725](https://github.com/brave/brave-browser/issues/22725) to address these issues.

Reply:
done